website/content/en/docs/cloud-providers/AWS.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Amazon Web Services (AWS)"
+linkTitle: "AWS"
+weight: 10
+---
+
+## Control Provisioning with Labels
+
+The [Provisioner CRD]({{< ref "provisioner-crd.md" >}}) supports defining
+instance types, node labels, and node taints.
+
+For certain well-known labels (documented below), Karpenter will provision
+nodes accordingly. For example, in response to a label of
+`topology.kubernetes.io/zone=us-east-1c`, Karpenter will provision nodes in
+that availability zone.
+
+Regarding AWS, 3 types of provisioning constraints are recognized: 
+- [Instance Type](#instance-type-allowlist)
+- [Node Labels](#node-lables-in-provisioner-spec) (e.g., Architecture, Capacity Type)
+- [Pod Labels](#pod-labels) (e.g., GPU)
+
+## Instance Type Allowlist

```

Comment:
I think allowlist is the wrong frame. Karpenter is built on a concepts of defaults (provisioner) + overrides (pod). A pod can specify a node selector for an instance type that is not in the provisioner's defaults.

Reply:
I'm confused - if a pod specifies an instance type not in karpenter's instance type list, karpenter will still provision it?

Reply:
Correct. The provisioner controls a set of defaults, but if the pod has a specific opinion, it will override the instance type and we'll use that.

Reply:
This definitely needs to be documented.